### ISSUE TITLE
Right-click opens and closes image panel, preventing context menu use

### ISSUE THREAD

**Issue by ShaunCassidyPoster** [fanfare/googleimagesrestored#39]
Right-clicking duplicates the left-click behavior of opening or closing the image panel. This prevents using the right-click context menu for some operations.

For example, when I have a set of image results I would like to quickly download with DownThemAll, I need to right-click on them and select "dta!OneClick" from the context menu, and I often need to do this for a number of images in a row or throughout the image results. (I usually download these quickly, serially, without opening the image panel at all--just right-clicking.)

With the current version of this add-on, that right-click opens or closes the image result each time, taking me away from the image results and focusing on the open image panel. If I go back to the results and try again, the right-click now closes the image panel. I would expect only left-click to open and close the panel.

Other than that, things are looking good. Thanks for making the add-on!

**Comment by fanfare** [comment]
Hi - i just tried right clicking on a bunch of stuff and nothing seemed different or abnormal at all. Can you give me more information -- what is your browser, what URL should i go to or search query to search for, and then what specific steps to take to reproduce what you're experiencing.  Sorry for the misunderstanding.

**Comment by SeanCassidyPoster** [comment]
Hi, thank you for looking at this. I have Firefox v52 and experience the behavior for any Google Images search, for example "[corn fritters](https://www.google.com/search?q=corn+fritters&source=lnms&tbm=isch&sa=X&ved=2ahUKEwi8_a6TpOjnAhWDFjQIHawrD1oQ_AUoAXoECBQQAw&biw=1920&bih=1002)."

I have also, I believe, ruled out my other add-ons that could conceivably interfere (uBlock Origin, TamperMonkey, GooglePanicImages) or userscripts like "Image Max Url" (used with Tamper Monkey) by turning them off and replicating my problem. The add-ons that allow downloading from the Google Images results page directly (by right-clicking) are DownThemAll and DownThemAll Anti-container.

One thing I tried that was interesting was to go to my search results, then after they load I activated an add-on called "Right to Click 2.9.6." This add-on is usually used for sites that attempt to block right-clicking with JavaScript. When I use it in this situation, the initial right-click still opens the image pane in the middle, but subsequent right-clicks do not close the pane, although they still switch the image in the pane. It's not really a workaround, but might give a clue about what's going on. I'm guessing "Right to Click" tries to inhibit any interference with the right-click event once it has been activated. However, I am not a programmer and am only guessing.

Thanks and let me know if I can provide any further info. I will keep trying to understand it here, and update if I learn anything new.
